frontend/src/pages/Account.tsx

```diff
@@ -0,0 +1,280 @@
+import {Trans, useTranslation} from "react-i18next";
+import {useEffect, useState} from "react";
+import {
+    AlertColor,
+    Box,
+    Container, Typography,
+} from "@mui/material";
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Unstable_Grid2';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import {DateField} from "@mui/x-date-pickers";
+import Chip from '@mui/material/Chip';
+import Paper from '@mui/material/Paper';
+import {styled} from '@mui/material/styles';
+import Alert from '@mui/material/Alert';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import CircularProgress from '@mui/material/CircularProgress';
+import Stack from '@mui/material/Stack';
+import {AdapterDayjs} from '@mui/x-date-pickers/AdapterDayjs';
+import {LocalizationProvider} from '@mui/x-date-pickers/LocalizationProvider';
+import dayjs from "dayjs";
+
+import {API_BASE_URL, signIn, signOut} from '../components/Api';
+import {Province, ChipData, chipColor, User} from "../components/Types";
+import httpClient from "../httpClient";
+import {GetProvincesWithNA} from "../components/Provinces";
+
+export const Account = () => {
+    const {t} = useTranslation();
+
+    const [province, setProvince] = useState<Province | null>(null);
+    const [chipData, setChipData] = useState<readonly ChipData[]>([]);
+
+    const [alert, setAlert] = useState(false);
+    const [alertType, setAlertType] = useState<AlertColor>("error");
+    const [alertContent, setAlertContent] = useState('');
+
+    const provinces: Province[] = GetProvincesWithNA();
+
+    const [user, setUser] = useState<User | null>(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const resp = await httpClient.get(API_BASE_URL + "/user_info");
+                setUser(resp.data);
+            } catch (error) {
+                console.log("Not authenticated");
+            }
+            setLoading(false);
+        })();
+    }, []);
+
+    let default_province: Province = {label: 'N/A', id: 'na', region: 'N/A', region_id: 'na'};
+    let default_dob = dayjs('2022-01-01');
+    let default_WCAID = "";
+    if (user != null) {
+        if (user.province != null) {
+            //set province in the combo box
+            for (let i = 0; i < provinces.length; i++) {
+                if (provinces[i].id === user.province) {
+                    default_province = provinces[i];
+                }
+            }
```

Comment:
I think you could `provinces.find(...)` the province? And get rid of the `let` at the same time. Something like
```
const defaultProvince = provinces.find(({ id }) => id === user?.province) || {label: 'N/A', id: 'na', region: 'N/A', region_id: 'na'}
```
Probably could do similar things with the dob and wcaid.